install.sh

```diff
@@ -81,17 +81,32 @@ sudo ln -s `pwd`/crew $CREW_PREFIX/bin
 mkdir $CREW_LIB_PATH/lib && cd $CREW_LIB_PATH/lib
 wget -N -c $URL/lib/package.rb
 wget -N -c $URL/lib/package_helpers.rb
-#create the device.json file
+
+#create or update the device.json file
+ruby_version='2.0.0p247'
```

Comment:
First, it's still consistent.  `install.sh` install ruby-2.0.0p347.  It is just there is a possibility to upgrade ruby to 2.4.0 after chromebrew installation.

If we decide to install the latest ruby in `install.sh`, we need to check whole installation process on every machines whenever we want to upgrade ruby.

Does this make sense?